services/yellow/src/api/Handlers/AuthorizeOrderCancellationFeeHandler.cs

```diff
@@ -29,7 +29,7 @@ protected override async Task<OrderCancellationFeeAuthorizationAcquired> Handle(
             if (authorizeResult.IsSuccess != true)
                 throw authorizeResult.Exception ?? new InvalidOperationException($"{nameof(authorizeResult)}");
             
-            _orderService.AddPaymentMethod(order, new OrderPaymentMethod(message.PaymentMethod.CreditCard.GetOrderCreditCard(authorizeResult.AuthorizePaymentToken)));
+            await AddPaymentMethodToOrder(_orderService, order, message.PaymentMethod.CreditCard, authorizeResult.AuthorizePaymentToken);
```

Comment:
Is there an alternative on having the `AddPaymentMethodToOrder` in the `AbstractHandler`? It would be better if it wasn't available to all the handlers.